<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>741</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="show">
      <property name="text">
       <string>显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnNormal">
      <property name="text">
       <string>正常</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnSmooth1">
      <property name="text">
       <string>光滑1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnSmooth2">
      <property name="text">
       <string>光滑2</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
